darts/timeseries.py

```diff
@@ -2766,7 +2792,7 @@ def stack(self, other: "TimeSeries") -> "TimeSeries":
         Return
         ------
         TimeSeries
-            A new multivariate TimeSeries instance.
+            A new multivariate and/or multi_sample TimeSeries instance.
```

Comment:
I think this change could be reverted.
```suggestion
            A new multivariate TimeSeries instance.
```